--- sapi/apache/php_apache.c.orig
+++ sapi/apache/php_apache.c
@@ -159,7 +159,7 @@
  */
 PHP_MINFO_FUNCTION(apache)
 {
-	char *apv = (char *) ap_get_server_version();
+	char *apv = (char *) ap_get_server_banner();
 	module *modp = NULL;
 	char output_buf[128];
 #if !defined(WIN32) && !defined(WINNT)
@@ -481,7 +481,7 @@
    Fetch Apache version */
 PHP_FUNCTION(apache_get_version)
 {       
-	char *apv = (char *) ap_get_server_version();
+	char *apv = (char *) ap_get_server_banner();
 
 	if (apv && *apv) {
 		RETURN_STRING(apv, 1);
--- ./sapi/apache2handler/php_functions.c.orig
+++ ./sapi/apache2handler/php_functions.c
@@ -320,7 +320,7 @@
 
 static char *php_apache_get_version()
 {
-	return (char *) ap_get_server_version();
+	return (char *) ap_get_server_banner();
 }
 
 /* {{{ proto string apache_get_version(void)
@@ -366,7 +366,7 @@
 	char *p;
 	server_rec *serv = ((php_struct *) SG(server_context))->r->server;
 #if !defined(WIN32) && !defined(WINNT)
-	AP_DECLARE_DATA extern unixd_config_rec unixd_config;
+	AP_DECLARE_DATA extern unixd_config_rec ap_unixd_config;
 #endif
 	
 	for (n = 0; ap_loaded_modules[n]; ++n) {
@@ -397,7 +397,7 @@
 	php_info_print_table_row(2, "Hostname:Port", tmp);
 	
 #if !defined(WIN32) && !defined(WINNT)
-	sprintf(tmp, "%s(%d)/%d", unixd_config.user_name, unixd_config.user_id, unixd_config.group_id);
+	sprintf(tmp, "%s(%d)/%d", ap_unixd_config.user_name, ap_unixd_config.user_id, ap_unixd_config.group_id);
 	php_info_print_table_row(2, "User/Group", tmp);
 #endif
 
